app/routes/profile.js

```diff
@@ -27,6 +27,21 @@ function ProfileHandler(db) {
         var bankAcc = req.body.bankAcc;
         var bankRouting = req.body.bankRouting;
 
+        // Fix for Section: ReDoS attack
+        // The following regexPattern that is used to validate the bankRouting number is insecure and vulnerable to
+        // catastrophic backtracking which means that specific type of input may cause it to consume all CPU resources
+        // with an exponential time until it completes
+        var regexPattern = /([0-9]+)+C/;
```

Comment:
A minor suggestion, can we require routing # to end with the number sign (#) instead of C?